java/arcs/android/storage/database/DatabaseImpl.kt

```diff
@@ -2399,7 +2384,6 @@ class DatabaseImpl(
       4 to CREATE_VERSION_3,
       5 to CREATE_VERSION_3,
```

Comment:
since you are here, could you change
4 to CREATE_VERSION_4,
5 to CREATE_VERSION_5

Reply:
No problem